docs/MigrationGuide.md

```diff
@@ -49,6 +49,11 @@ val source = new KafkaAvroSourceFactory(schemaRegistryProvider, processObjectDep
 val avroFixedSourceFactory = FixedKafkaAvroSourceFactory[GenericData.Record](processObjectDependencies)
 ```
 
+## In version 0.1.2 (not released yet)
```

Comment:
Do we want it in 0.1.2? Maybe in 0.2.0? It's not so badly needed, and it's not fully backwards compatible...

Reply:
IMO it is quite badly needed, because it is fix for two bugs:
- we suggest methods that can't be used
- we accept something in expressions and it blow up in runtime.
Compatibility is broken only on very advanced usages, when you extract scala map from `TypedMap` (i can add such extractor BTW but IMO it will mess a code a bit)